odoo/addons/base/tests/test_ir_actions.py

```diff
@@ -315,6 +316,15 @@ def test_80_permission(self):
         with self.assertRaises(AccessError), mute_logger('odoo.addons.base.models.ir_actions'):
             self_demo.with_context(self.context).run()
 
+    def test_90_convert_to_float(self):
+        #make sure eval_value convert the value into float for float-type fields
+        self.action.write({
+            'state': 'object_write',
+            'fields_lines': [Command.create({'col1': self.res_partner_credit_limit_field.id, 'value':'20.99'})],
```

Comment:
Still a missing space
```suggestion
            'fields_lines': [Command.create({'col1': self.res_partner_credit_limit_field.id, 'value': '20.99'})],
```